### PR TITLE
move to k8s.io v1.20.5

### PR DESCRIPTION
**Pull request by mythi**
.5 was released just after I updated to .4. Move to .5 still before the release.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,42 +13,42 @@ require (
...
-	k8s.io/kubelet => k8s.io/kubelet v0.20.4
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.4
-	k8s.io/metrics => k8s.io/metrics v0.20.4
-	k8s.io/mount-utils => k8s.io/mount-utils v0.20.5-rc.0
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.4
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.20.4
-	k8s.io/sample-controller => k8s.io/sample-controller v0.20.4
+	k8s.io/api => k8s.io/api v0.20.5
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.5
+	k8s.io/apimachinery => k8s.io/apimachinery v0.20.6-rc.0
```
Comment (kad) [inline go.mod:30]: there is v0.20.5 tag. is that intentional 0.20.6-rc?

Comment (mythi) [reply go.mod:30]: this comes from the script we are using (DEVEL.md).

File: go.mod [hunk 2]
```diff
@@ -13,42 +13,42 @@ require (
...
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.5
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.5
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.5
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.5
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.5
+	k8s.io/kubectl => k8s.io/kubectl v0.20.5
+	k8s.io/kubelet => k8s.io/kubelet v0.20.5
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.5
+	k8s.io/metrics => k8s.io/metrics v0.20.5
+	k8s.io/mount-utils => k8s.io/mount-utils v0.20.6-rc.0
```
Comment (bart0sh) [inline go.mod:50]: This one is also 0.20.6-rc.0. Can you explain why it got here?

Comment (mythi) [reply go.mod:50]: @bart0sh @kad This is the reason why we get those:
```go mod download -json k8s.io/apimachinery@kubernetes-1.20.5
{
	"Path": "k8s.io/apimachinery",
	"Version": "v0.20.6-rc.0",
	...
}```

the script picks up that `Version`

**Review by bart0sh — DISMISSED** [review]
/lgtm

**Comment by mythi** [comment]
@bart0sh the build is passing now..

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**